internal/tests/cli/boundary/target_host_sources.bats

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env bats
```

Comment:
Is there some overlap with this test file and some the existing bats tests? I'm struggling to identify the differentiator here. Is it the validating of the host sources on the target? Could that just be added to `targets.bats`?

Reply:
I also realized some overlap, but this test file is similar to `target_credential_sources.bats` where the test focuses on host sources. It could be expanded to cover even more use cases. `targets.bats` may not be focused on all the possible use cases for `host-source`.